scripts/meson.build

```diff
@@ -17,4 +17,6 @@ abi = shared_library(
   abi_src, stat_src, common_src,
   name_prefix : '',
   dependencies : [prog_python.dependency(embed : true), common_dep, nl_nf_3],
+  install: true,
+  install_dir : prog_python.get_install_dir()
```

Comment:
This is not the way to go. `abi` needs to be converted into a proper python module, as this just dumps it into `site-packages` where it doesn't belong.

Reply:
I am bit lost here because we speaking about C-library imported into python script . So what would be "proper python module" ? You want get rid of C-library and migrate C code into python flavor ? It will not be accepted because this C code is also used in C part of phoebe

Reply:
@asmorodskyi I think @dcermak is referring to the usage of what he suggested in the issue itself. 
See his last comment on https://github.com/SUSE/phoebe/issues/38

Reply:
@mvarlese  yeah I have seen that comment. Also I had private chat with @dcermak.  So while my approach is working I am accepting the fact that it is not "canonical" . On the other hand I don't clear understanding how to proceed with [canonical](https://mesonbuild.com/Python-module.html#extension_module) approach. Also from what I know from @shunghsiyu  this feature (collect_stats.py ) suppose to get major changes in foreseen future anyway. Summing up all this I suggest two options : 
1. Merge this approach ( yes having C library in python extension is not nice but not bring any real problems AFAIK ) 
2. Leave as it is and close issue  https://github.com/SUSE/phoebe/issues/38

Reply:
@dcermak would you mind suggesting a patch showing how you see this being implemented, please?

Reply:
From a _practical_ perspective both `shared_library()` that's used in this pull request and `extension_module()` suggested by Dan should result the in same `.so` file. A shared library containing some specific set of symbols (which `_phoebe.abi3.so` does) is a valid form of Python module.

That said I do prefer the [canoical](https://mesonbuild.com/Python-module.html#extension_module) approach; this way it is more clear that the result is a Python module, not some plain shared library that is expected to work non-Python process.

Something like this may work

```diff
- abi = shared_library(...)
+ abi = prog_python.extension_module(...)
```

Reply:
@asmorodskyi would you mind seeing if the recommendation from @shunghsiyu fixes the issue you pointed out, please?

Reply:
but there quite some failures in CI ... will check that later ...

Reply:
@asmorodskyi the failure is caused by the CI/CD not finding the library _phoebe. I suppose it is caused by the fact that @shunghsiyu used to set the PYTHONPATH to build/scripts prior to your patch (see https://github.com/SUSE/phoebe/blob/aa821da/tests/meson.build#L39). You may need to change/remove that now.

Reply:
> 
> 1. Merge this approach ( yes having C library in python extension is not nice but not bring any real problems AFAIK )
> 2. Leave as it is and close issue  #38

@mvarlese , @shunghsiyu , @dcermak  - I think problem which we trying to solve in #38 smaller than all things which are coming out while we doing this. I more and more looking into direction of second point ...